core/game_object.lua

```diff
@@ -1939,40 +1939,57 @@ Set `prefix_config.key = false` on your object instead.]]):format(obj.key), obj.
     ----- API CODE GameObject.PokerHand
     -------------------------------------------------------------------------------------------------
 
+    SMODS.PokerHandParts = {}
+    SMODS.PokerHandPart = SMODS.GameObject:extend {
+        obj_table = SMODS.PokerHandParts,
+        obj_buffer = {},
+        required_params = {
+            'key',
+            'func',
+        },
+        inject_class = function() end,
+    }
+    local handlist = G.handlist
+    G.handlist = {}
     SMODS.PokerHands = {}
     SMODS.PokerHand = SMODS.GameObject:extend {
         obj_table = SMODS.PokerHands,
         obj_buffer = {},
         required_params = {
             'key',
-            'above_hand',
             'mult',
             'chips',
             'l_mult',
             'l_chips',
             'example',
+            'evaluate'
         },
         order_lookup = {},
+        max_order = { value = 0 },
         visible = true,
         played = 0,
         played_this_round = 0,
         level = 1,
-        class_prefix = 'h',
```

Comment:
I support this, since it was confusing. Reminder that we should warn people about this